buildSrc/src/main/groovy/org/gradle/build/ReleasedVersion.groovy

```diff
@@ -23,7 +23,7 @@ import org.gradle.util.GradleVersion
 import java.text.SimpleDateFormat
 
 @Canonical
-class ReleasedVersion {
+class ReleasedVersion implements Serializable {
```

Comment:
Why does `ReleasedVersion` need to be serializable?

Reply:
I believe I wanted to use it as `@Input` somewhere. I am going to remove it for now.